pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```diff
@@ -1,86 +1,55 @@
 {
   lib,
   stdenv,
-  stdenvNoCC,
   fetchFromGitHub,
   rustPlatform,
-  buildGoModule,
-  nix-update-script,
-  modrinth-app-unwrapped,
   cacert,
   cargo-tauri,
-  desktop-file-utils,
-  esbuild,
   darwin,
-  jq,
+  desktop-file-utils,
   libsoup,
-  moreutils,
-  pnpm_8,
   nodejs,
   openssl,
   pkg-config,
+  pnpm_9,
+  turbo,
   webkitgtk,
 }:
-rustPlatform.buildRustPackage {
+
+let
+  pnpm = pnpm_9;
+in
+rustPlatform.buildRustPackage rec {
   pname = "modrinth-app-unwrapped";
-  version = "0.7.1";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "modrinth";
-    repo = "theseus";
-    rev = "v${modrinth-app-unwrapped.version}";
-    hash = "sha256-JWR0e2vOBvOLosr22Oo2mAlR0KAhL+261RRybhNctlM=";
+    repo = "code";
+    rev = "a0bd011b808cdc998ef27960f610a8d99550c914";
+    hash = "sha256-zpFJq7if5gOx7jvwpE73lqH4Vpif0MJMPIGsgtThKVk=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "tauri-plugin-single-instance-0.0.0" = "sha256-Mf2/cnKotd751ZcSHfiSLNe2nxBfo4dMBdoCwQhe7yI=";
+      "sqlx-0.8.0-alpha.0" = "sha256-M1bumCMTzgDcQlgSYB6ypPp794e35ZSQCLzkbrFV4qY=";
+      "tauri-plugin-single-instance-0.0.0" = "sha256-DZWTO2/LevbQJCJbeHbTc2rhesV3bNrs+BoYm2eMakA=";
     };
   };
 
-  pnpm-deps = stdenvNoCC.mkDerivation (finalAttrs: {
-    pname = "${modrinth-app-unwrapped.pname}-pnpm-deps";
-    inherit (modrinth-app-unwrapped) version src;
-    sourceRoot = "${finalAttrs.src.name}/theseus_gui";
-
-    dontConfigure = true;
-    dontBuild = true;
-    doCheck = false;
-
-    nativeBuildInputs = [
-      cacert
-      jq
-      moreutils
-      pnpm_8
-    ];
-
-    # https://github.com/NixOS/nixpkgs/blob/763e59ffedb5c25774387bf99bc725df5df82d10/pkgs/applications/misc/pot/default.nix#L56
-    installPhase = ''
-      export HOME=$(mktemp -d)
-
-      pnpm config set store-dir "$out"
-      pnpm install --frozen-lockfile --ignore-script --force
-
-      # remove timestamp and sort json files
-      rm -rf "$out"/v3/tmp
-      for f in $(find "$out" -name "*.json"); do
-        sed -i -E -e 's/"checkedAt":[0-9]+,//g' $f
-        jq --sort-keys . "$f" | sponge "$f"
-      done
-    '';
-
-    dontFixup = true;
-    outputHashMode = "recursive";
-    outputHash = "sha256-g/uUGfC9TQh0LE8ed51oFY17FySoeTvfaeEpzpNeMao=";
-  });
+  pnpmDeps = pnpm.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-xD04CD3fevIKZoXpjMo3tRF2r7K2IiMh1cNS7/FNx2I=";
+  };
```

Comment:
Please rebase and regenerate hashes.

See https://github.com/NixOS/nixpkgs/pull/349093#issuecomment-2421873637